<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookMessage</class>
 <widget class="QDialog" name="LookMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="contactLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>朗读</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
